nucypher/blockchain/eth/clients.py

```diff
@@ -97,13 +98,22 @@ def _add_default_middleware(self):
         chain_id = self.chain_id
         is_poa = chain_id in POA_CHAINS
 
-        self.log.debug(
+        self.log.info(
             f"Blockchain: {self.chain_name} (chain_id={chain_id}, poa={is_poa})"
         )
         if is_poa:
             # proof-of-authority blockchain
             self.log.info("Injecting POA middleware at layer 0")
-            self.inject_middleware(geth_poa_middleware, layer=0, name="poa")
```

Comment:
Wait, but didn't we decide to always inject the POA middleware?

Reply:
We did not end up doing that because we were concerned about the possible side effects of always adding the middleware regardless of POA or not.